sql/dj_setup_database.in

```diff
@@ -327,6 +327,18 @@ upgrade)
 		echo "You may also remove those users from MySQL."
 	fi
 	read_dbpasswords
+
+	# Check if we need to upgrade the Doctrine migrations table
+	if ! echo "SHOW CREATE TABLE \`doctrine_migration_versions\`" | mysql "$DBNAME" >/dev/null 2>&1; then
+		symfony_console doctrine:migrations:sync-metadata-storage -n
+		# shellcheck disable=SC2016
+		echo 'INSERT INTO `doctrine_migration_versions`
+			(version, executed_at, execution_time)
+			SELECT concat("DoctrineMigrations\\\\Version", version), executed_at, 1
+			FROM migration_versions;' | mysql "$DBNAME"
+		 echo "DROP TABLE \`migration_versions\`" | mysql "$DBNAME"
```

Comment:
I think the article you linked advised to do also check with `doctrine:migrations:status`

Reply:
I tried that. That only says “you need to apply all 44 migrations”. Didn’t really help me

Reply:
Ok, for the set I was working with it gave an error. So I think testing that the exit code is `0` should be considered but this is only relevant when people have added their own migrations which we maybe don't need to anticipate.

Reply:
Even if people added their own migration we copy it over to the new table. I think this is not too relevant.